src/services/CommonService.ts

```diff
@@ -93,6 +93,35 @@ export interface CompanyType {
   subNavigation: SubNavigationType[]
 }
 
+export interface StandardLibraryType {
+  rows: StdRows[]
+  useCases: StdObject[]
+  capabilities: StdObject[]
+  roles: StdObject[]
+  status: StdObject[]
+  typesOfDocuments: StdObject[]
+}
+
+export interface StdRows {
+  uid: number
+  name: string
+  title: string
+  releaseOfDocument: string
+  standardizationCandidate: string
+  description: string
+  download: string
+  roles: number[]
+  capabilities: number[]
+  useCases: number[]
+  typesOfDocuments: number[]
+  status: number[]
+}
+export interface StdObject {
+  uid: number
+  name: string
+  title: string
+}
+
```

Comment:
CommonService is becoming a collection of random code for all kinds of stuff - a service with that name should never have been created because the name suggests to put anything in there. Please create a feature with an standardLibraryApiSlice as we do with any API.